src/main/java/com/example/sixthmafiabot/repository/PlayerRepository.java

```diff
@@ -1,22 +1,56 @@
 package com.example.sixthmafiabot.repository;
 
+import com.example.sixthmafiabot.DTO.CreatePlayerDTO;
+import com.example.sixthmafiabot.DTO.UpdatePlayerDTO;
 import com.example.sixthmafiabot.models.Game;
 import com.example.sixthmafiabot.models.Player;
 import com.example.sixthmafiabot.models.User;
-import org.springframework.data.repository.Repository;
+import com.example.sixthmafiabot.repository.Abstract.BaseRepository;
+import com.example.sixthmafiabot.repository.Abstract.SpringRepositoryImplementations.SpringPlayerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Repository;
 
-import java.util.concurrent.CompletableFuture;
+@Repository
+public class PlayerRepository extends BaseRepository {
 
-public interface PlayerRepository extends Repository<Player, Long> {
+    @Autowired
+    SpringPlayerRepository springPlayerRepository;
 
-    @Async("asyncExecutor")
-    void save(Player player);
+    public Player create(CreatePlayerDTO playerDTO){
 
-    @Async("asyncExecutor")
-    CompletableFuture<Player> getPlayerByUser(User user);
+        Player player = modelMapper.map(playerDTO, Player.class);
+
+        springPlayerRepository.save(player);
+
+        return player;
+    }
+
+    public Player update(Player playerInDatabase, UpdatePlayerDTO updatePlayerDTO){
+
+        playerInDatabase.setIsAlive(updatePlayerDTO.isAlive());
```

Comment:
why do you use method instead of mapper?

Reply:
+